lib/karafka/lago_monitor.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Karafka::LagoMonitor < ::Karafka::Instrumentation::Monitor
+  TRACEABLE_EVENTS = %w[
+    consumer.consumed
+  ].freeze
+
+  def instrument(event_id, payload = EMPTY_HASH, &block)
```

Comment:
Where is `EMPTY_HASH` defined? I don't see it in our codebase so I assume we're using it from another gem. We should probably have one. I needed it for permissions too

https://github.com/getlago/lago-api/blob/bc3ebfc462b9e2d6e7dbf2bba5b7da2dce9cad29/app/models/permission.rb#L10

Reply:
It's part of the gem (https://github.com/karafka/karafka/blob/3fd32d63b2f2d78321055c3e8b2b22d0e7ca3fbd/lib/karafka/processing/inline_insights/tracker.rb#L37)
I just used an example from the Karafka documentation